x-pack/plugins/cases/public/components/all_cases/selector_modal/use_cases_add_to_existing_case_modal.test.tsx

```diff
@@ -89,4 +125,47 @@ describe('use cases add to existing case modal hook', () => {
       })
     );
   });
+
+  it('should call postComment when a case is selected', async () => {
```

Comment:
Could you please add a test that tests that the toast has been called successfully?

Reply:
Added